tutorials/hist/ZoomHistogram.C

```diff
@@ -0,0 +1,38 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Changing the Range on the X-Axis of a Histogram
+///
+/// Image produced by `.x ZoomHistogram.C`
+///
+/// This demonstrates how to zoom into a Histogram by
+/// changing the range on one of the axes (or both).
+///
+/// \macro_image
+/// \macro_code
+///
+/// \author Advait Dhingra
+
+void ZoomHistogram() {
+
+   TH1F *norm = new TH1F("Normal Histogram", "Normal Histogram", 100, 0, 100);
+
+   TRandom2 rand;
+
+   for (int i = 0; i < 100; ++i) {
+     Double_t j = rand.Gaus(50, 10);
+     norm->Fill(j);
```

Comment:
```suggestion
     Double_t x = rand.Gaus(50, 10);
     norm->Fill(x);
```
"j" sounds like an inner index (compared to "i"), where really it's a value on the x-axis.

Reply:
Good point